installer/functions

```diff
@@ -239,3 +239,60 @@ websocketcommand() {
         echo "EError timeout"
     fi
 }
+
+# Check space requirements on CHROOTS partition
+# $1: the partition's minimum size threshold
+# $2: the partition's minimum available space threshold
+check_space() {
+    local part=${CHROOTS:-/usr/local}
+    df -m $part | awk 'FNR == 2 {print $2,$4,$6}' |\
+    while read -r size avail mp; do
```

Comment:
This doesn't need to be a while loop, since you always "return" after one round.  Just start a `{ }` block after the `|` on the upper line.